### PR TITLE
Namespaced all classes.

### PR DESCRIPTION
**Pull request by taggon**
Now all components are autoloaded.
Plus, I removed some empty files and folders to make the codebase neat.

### CODE REVIEW

File: server/class-component-themes-builder.php [hunk 1]
```diff
@@ -120,6 +123,8 @@ private function expand_config_partials( $component_config, $partials ) {
 			if ( isset( $partials[ $partial_key ] ) ) {
 				return $partials[ $partial_key ];
 			}
+			print_r( $partials );
+			echo  '['.$partial_key .']<br>';
 			throw new Exception( 'No partial found matching ' . $partial_key );
```
Comment (sirbrillig) [inline server/class-component-themes-builder.php]: If using namespaces, you'll have to change `Exception` to `\Exception`

File: server/class-component-themes-builder.php [hunk 2]
```diff
@@ -120,6 +123,8 @@ private function expand_config_partials( $component_config, $partials ) {
 			if ( isset( $partials[ $partial_key ] ) ) {
 				return $partials[ $partial_key ];
 			}
+			print_r( $partials );
```
Comment (sirbrillig) [inline server/class-component-themes-builder.php]: Probably left-over 😉

**Comment by sirbrillig** [comment]
This is awesome, and I *really* want to use it, but I'm worried about compatibility of using namespaces. This is because the [minimum PHP version](https://wordpress.org/about/requirements/) for WP is 5.2.4 and namespaces weren't introduced until 5.3. In fact, I recently removed all the namespaces  in 2f1e0e2fdc9ab1847718ee54f9dc703df7031975. 

That said, it makes everything *so* much cleaner, I wonder if we could bump the minimum version for this plugin to 5.3. Worth a discussion, certainly.

File: server/class-component-themes-builder.php (hunk 2 above)
Comment (taggon) [reply server/class-component-themes-builder.php]: My bad. 😱

**Comment by taggon** [comment]
> the minimum PHP version for WP is 5.2.4 and namespaces weren't introduced until 5.3.

That's so frustrating. 😭

**Pull request closed without merging**